python_modules/dagster/dagster/_core/execution/backfill.py

```diff
@@ -126,6 +126,44 @@ def is_valid_serialization(self, workspace: IWorkspace) -> bool:
         else:
             return True
 
+    def get_num_partitions_targeted_by_asset(self, workspace: IWorkspace) -> Mapping[AssetKey, int]:
```

Comment:
Maybe worth combining these two new methods into a single one that returns a single object with all the data, so that we don't need to deserialize the backfill data and build the asset graph both times?